lib/usd/translators/shading/usdStandardSurfaceReader.cpp

```diff
@@ -82,6 +86,59 @@ PxrUsdTranslators_StandardSurfaceReader::PxrUsdTranslators_StandardSurfaceReader
 {
 }
 
+PxrUsdTranslators_StandardSurfaceReader::~PxrUsdTranslators_StandardSurfaceReader()
+{
+    // Expand Opacity from R to RGB if necessary.
+    if (!_standardSurfaceObj.isNull()) {
```

Comment:
Did we do something like this before? It doesn't look kosher to rely on the lifetime of an object to fix up the connectivity.

Reply:
Made everything explicit.